server/devdocs/index.js

```diff
@@ -59,18 +61,17 @@ function listDocs( filePaths ) {
 				title: doc.title,
 				snippet: defaultSnippet( doc )
 			};
-		} else {
-			return {
-				path: path,
-				title: 'Not found: ' + path,
-				snippet: ''
-			};
 		}
+		return {
+			path: path,
+			title: 'Not found: ' + path,
+			snippet: ''
+		};
 	} );
 	return results;
 }
 
-/**
```

Comment:
It'd be nice if we had properly formatted JSDoc for these functions, though obviously not existing previously and currently a lint warning with the double-asterisk.